content/feature/Balance Sheet Implementation - Hellhound14.md

```diff
@@ -30,11 +28,8 @@ UPDATE public.table1
 
 
 ### Code Breakdown and Explanation
+The `UPDATE` statement updates the values of the “Monthly Earnings ($)” column by subtracting the values of the “Loss ($)” column from the “Profit ($)” one.
 
-The `CREATE TABLE` statement creates a new table with the name `“table1”` and adds the columns `“Date”, “Profit ($)”, “Loss ($)” and “Earnings ($)”` of the defined data types.
-
-In the newly formed “table1”, using the statement `INSERT INTO` new `VALUES` are added.
-
-Finally, the `UPDATE` statement updates the values of the `“Earnings ($)”` column by subtracting the values of the `“Loss ($)”` column from the `“Profit ($)”` one.
+The `Lag` function then creates a new column from the values of the “Monthly Earnings ($)” column just placing them row before.

```

Comment:
I don't think this is a very good explanation of Lag can you improve here a bit?